permissions/plugin-vk-notifier.yml

```
@@ -0,0 +1,11 @@
+---
+name: "vk-notifier"
+github: "jenkinsci/vk-notifier-plugin"
+issues:
+- github: "jenkinsci/vk-notifier"
+paths:
+- "org/jenkins-ci/plugins/vk-notifier"
```

https://github.com/jenkinsci/vk-notifier-plugin/blob/9e1cb999caea84ac3c6c57f7cd4b920f5a7a2ae8/build.gradle#L9
```suggestion
- "io/jenkins/plugins/vk-notifier"
```